kube/bundle.yaml

```diff
@@ -117,6 +117,11 @@ data:
       listenAddress:
         type: "config"
         value: "0.0.0.0:7201"
+      local:
```

Comment:
is this correct for kube?

Reply:
Yeah - @schallert said to update here as well